feature-libs/asm/customer-360/components/asm-customer-promotion-listing/asm-customer-promotion-listing.component.ts

```diff
@@ -29,9 +30,16 @@ export class AsmCustomerPromotionListingComponent {
   @Input() showAlertForApplyAction: boolean | null;
   @Input() showRemoveButton: boolean;
   @Input() showApplyButton: boolean;
+  @Input() isCustomerCoupon: boolean;
   @Output() apply = new EventEmitter<PromotionListEntry>();
   @Output() remove = new EventEmitter<PromotionListEntry>();
   @Output() removeAlert = new EventEmitter();
   @Output() removeAlertForApplyAction = new EventEmitter();
+  @Output() refreshCustomerCouponList = new EventEmitter<{
+    assignable: boolean | undefined;
```

Comment:
```suggestion
    assignable?: boolean;
```

Reply:
the code has been refactored. Please check the latest code.